content/index.html

```diff
@@ -116,39 +116,38 @@ <h1><i class="fa fa-github"></i> Open Source</h1>
 
   <div class="container">
       <div class="row logos">
-          <a href="http://argus-sec.com/"><img src="assets/company-logos/Argus.png"/></a>
-          <a href="http://www.boxever.com/"><img src="assets/company-logos/Boxever.png"/></a>
-          <a href="http://www.branch.io/"><img src="assets/company-logos/Branch.png"/></a>
+          <a href="https://argus-sec.com/"><img src="assets/company-logos/Argus.png"/></a>
+          <a href="https://www.boxever.com/"><img src="assets/company-logos/Boxever.png"/></a>
+          <a href="https://www.branch.io/"><img src="assets/company-logos/Branch.png"/></a>
           <a href="https://www.cesanta.com/"><img src="assets/company-logos/cesanta.png"/></a>
           <a href="https://www.coreos.com/"><img src="assets/company-logos/CoreOS.png"/></a>
-          <a href="http://www.crononauta.com/"><img src="assets/company-logos/crononauta.png"/></a>
+          <a href="https://www.crononauta.com/"><img src="assets/company-logos/crononauta.png"/></a>
           <a href="https://www.digitalocean.com/"><img src="assets/company-logos/digitalocean.png"/></a>
           <a href="https://www.docker.com/"><img src="assets/company-logos/Docker.png"/></a>
-          <a href="http://www.ericsson.com/"><img src="assets/company-logos/Ericsson.png"/></a>
-          <a href="http://www.eurotech.com/"><img src="assets/company-logos/Eurotech.png"/></a>
+          <a href="https://www.ericsson.com/"><img src="assets/company-logos/Ericsson.png"/></a>
+          <a href="https://www.eurotech.com/"><img src="assets/company-logos/Eurotech.png"/></a>
           <a href="https://www.fosdem.org/"><img src="assets/company-logos/Fosdem.png"/></a>
           <a href="https://giantswarm.io/"><img src="assets/company-logos/giantswarm.png"/></a>
+	  <a href="https://grafana.com"><img src="assets/company-logos/grafana.png"/></a>
```

Comment:
This one's indented inconsistently.

Reply:
My naughty Emacs introduced a tab here. Untabify helped…